src/hotspot/share/cds/cdsConfig.hpp

```diff
@@ -30,7 +30,9 @@
 
 class CDSConfig : public AllStatic {
 #if INCLUDE_CDS
-  static bool _is_dumping_dynamic_archive;
+  static bool     _is_dumping_dynamic_archive;
```

Comment:
Nit: Extra spaces

Reply:
The alignment is the same as the is_dumping_xxx()/enable_dumping_xxx() functions below, so it's easier to read the "dumping" and "loading" part of the names.

Reply:
Sorry that is not a valid reason IMO. It's bad enough when people want the start of names to line up, never-mind the middle of them!

Reply:
That's my personal preference. Many people align the code differently. Are there any rules in the code guide about code alignments?

Reply:
Nope no rules as such.